README.md

```diff
@@ -1 +1,15 @@
-# sandbox-exadel-2022
\ No newline at end of file
+# **Welcome to budget keeping app !**
+
+---
+
+## `NPM RUN DEV`
```

Comment:
add `npm install` as well, and make it as a list of steps to do (https://www.markdownguide.org/basic-syntax/#lists-1)


also its better to write commands like that
```suggestion
> npm run dev
```
or just
```suggestion
`npm run dev`
```